### PR TITLE
alexander95015's Slack Cheat Sheet

### PR DESCRIPTION
**Pull request by alexander95015**
# Goodie Cheat Sheet Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**
Provide a cheat sheet for Slack users.

**What is the data source for your Instant Answer? (Provide a link if possible)**
https://slack.zendesk.com/hc/en-us/articles/201374536-Slack-keyboard-shortcuts

**Why did you choose this data source?**
It's Slack's official documentation.

**Are there any other alternative (better) data sources?**
Probably not.

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Slack users.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
https://duck.co/ideas/idea/6330/slack-cheat-sheet

**Are you having any problems? Do you need our help with anything?**

**Where did you hear about DuckDuckHack? (For first time contributors)**

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screen shot 2015-08-30 at 11 21 04 am

https://duck.co/ia/view/slack_cheat_sheet

### CODE REVIEW

**Comment by jagtalon** [comment]
@alexander95015 this is perfect since we just moved to Slack! Please claim the Idea in https://duck.co/ideas/idea/6330/slack-cheat-sheet!

Another thing is that we're currently not trying to put both Mac shortcuts and Windows (and other systems) on the same cheat sheet so for now use `[Ctrl]` instead of the `Cmd` key. We'll make an update on the cheat sheet system to be able to switch depending on the system but we don't have that yet. We're doing this because it tends to get crowded visually.

**Comment by alexander95015** [comment]
@jagtalon Someone else seems to have claimed it. I'll get rid of the command keys.

File: share/goodie/cheat_sheets/json/slack.json [hunk 1]
```diff
@@ -0,0 +1,168 @@
+{
+    "id": "slack_cheat_sheet", 
+
+    "name": "Slack",
+
+    "description": "Work chat", 
+    
+    "metadata": { 
+        "sourceName": "Slack keyboard shortcuts",
+        "sourceUrl": "https://slack.zendesk.com/hc/en-us/articles/201374536-Slack-keyboard-shortcuts"
```
Comment (jagtalon) [inline share/goodie/cheat_sheets/json/slack.json]: Is it just me or is https://slack.zendesk.com/hc/en-us/articles/201374536-Slack-keyboard-shortcuts down?

**Comment by jagtalon** [comment]
Ok IA page is here: https://duck.co/ia/view/slack_cheat_sheet

**Comment by alexander95015** [comment]
@jagtalon I can't seem to edit the IA page for some reason.

**Comment by jagtalon** [comment]
@alexander95015 you should be able to now. :)

File: share/goodie/cheat_sheets/json/slack.json [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+                "val": "Open Team Directory"                   
+            },
+            {
+                "key": "[Ctrl] [Shift] [S]",
+                "val": "Open Starred Items"                   
+            }
+        ],
+        "Switching teams": [
+            {
+                "key": "[Ctrl] [Shift] [left square bracket]",
```
Comment (jagtalon) [inline share/goodie/cheat_sheets/json/slack.json]: Sholdn't this be `[Ctrl] [Shift] [\\[]]`?

File: share/goodie/cheat_sheets/json/slack.json [hunk 3]
```diff
@@ -0,0 +1,168 @@
+{
+    "id": "slack_cheat_sheet", 
+
+    "name": "Slack",
+
+    "description": "Work chat", 
+    
+    "metadata": { 
+        "sourceName": "Slack keyboard shortcuts",
```
Comment (jagtalon) [inline share/goodie/cheat_sheets/json/slack.json]: Probably better to just call it "Slack"

**Comment by jagtalon** [comment]
@alexander95015 thanks for the fixes. :+1:

**Comment by moollaza** [comment]
@alexander95015 congrats! It's live: https://duckduckgo.com/?q=slack+cheat+sheet&t=ffab&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**